packages/office-ui-fabric-react/src/components/Fabric/Fabric.styles.ts

```diff
@@ -16,10 +16,12 @@ export interface IFabricClassNames {
 export const getStyles = (props: IFabricStyleProps): IFabricStyles => {
   const { theme, className, isFocusVisible, applyTheme } = props;
   const classNames = getGlobalClassNames(GlobalClassNames, theme);
+  const focusVisibility = isFocusVisible ? 'Visible' : 'Hidden';
```

Comment:
Just to clarify, we're currently just disabling focus altogether in the alternate case but now instead of that we're toggling to hidden?

Reply:
Currently, the "visible" class is added on keystroke detection (of a defined set of "directional" keys). That same class is then removed on mouse-click detection. That's it. So, there is actually no "hidden" concept in Fabric currently; it is only _implicitly implied_ by the absence of the "visible" class. This is a bit unfortunate, as that absence could also be due to other factors. More on this in the PR description. Alternating between the "hidden" and "visible" eliminates ambiguity and gives users more flexibility in how to define focus indication. (Mainly: No more need to disable focus indication by default.)